src/tools/clippy/clippy_lints/src/attrs.rs

```diff
@@ -1,15 +1,20 @@
 //! checks for attributes
 
-use clippy_utils::diagnostics::{span_lint, span_lint_and_help, span_lint_and_sugg, span_lint_and_then};
+use clippy_utils::diagnostics::{
```

Comment:
Please don't commit format changes to Clippy. This messes up the sync completely.

Reply:
Gotcha. Will revert the formatting.